tools/sponge/preproc_scripts/get_so_monthly.csh

```diff
@@ -0,0 +1,31 @@
+#!/bin/tcsh
+#SBATCH --ntasks=1
+#SBATCH --job-name=fill_glorys_arctic
+#SBATCH --time=2880
+#SBATCH --partition=batch
+
+# Usage: sbatch fill_glorys_nn_monthly.sh <YEAR> <MONTH>
+
+module load cdo
+module load nco/5.0.1
+module load gcp
+
+set year=$1
+set month=`printf "%02d" $2`
+
+set apath='/archive/Theresa.Morrison/datasets/glorys/GLOBAL_MULTIYEAR_PHY_001_030/monthly/so'
+mkdir ${apath}/so_${year}_${month}
+
+set day=1
+foreach filename (/uda/Global_Ocean_Physics_Reanalysis/global/daily/so/${year}/so_mercatorglorys12v1_gl12_mean_${year}${month}*.nc)
+  echo $filename 
+  set short_name='so_arctic_'$day
+  ncks -d latitude,39.,91. --mk_rec_dmn time $filename ${apath}/so_${year}_${month}/${short_name}'_bd.nc'
+  cdo -setreftime,1993-01-01,00:00:00,1day ${apath}/so_${year}_${month}/${short_name}'_bd.nc' ${apath}/so_${year}_${month}/${short_name}'.nc'
+  rm -f ${apath}/so_${year}_${month}/${short_name}'_bd.nc'
+  set day = `expr $day + 1`
+  echo $day
+end
+ncra -O --cnk_plc=r1d --cnk_dmn=time,1  ${apath}/so_${year}_${month}/so_arctic_*.nc ${apath}/GLORYS_so_arctic_${year}_${month}.nc
```

Comment:
I know this is doing the time average, but for clarity can you add a comment stating that, and also describing what `--cnk_plc=r1d --cnk_dmn=time,1` is doing?

Reply:
This was meant to help the averaging be faster, but it's not neede any more.